templates/forms/layouts/field.html.twig

```diff
@@ -1,23 +1,23 @@
 {% block field %}
 <div class="form-field {{ layout_form_field_outer_classes|trim -}} {{- form_field_outer_core|trim -}}">
   {% block contents %}
-    {% if show_label %}
-      <div class="{{- layout_form_field_outer_label_classes -}}">
-        {{- form_field_toggleable -}}
-        <label class="{{ layout_form_field_label_classes }}{{ form_field_label_trim }}" {% if field.id %}for="{{ form_field_for }}"{% endif %}>
-          {%- block label -%}
-            {%- if form_field_help -%}
-              <span class="tooltip" data-tooltip="{{ form_field_help|e }}">{{ form_field_label|raw }}</span>
```

Comment:
Why was the escaping removed here?

Reply:
I don't like us using `|raw` without concent from the person who writes the form. Either the property should be called `label_html` or it should indicate it in some other way, maybe something like

https://github.com/getgrav/grav-plugin-form/pull/474#pullrequestreview-565195428

but with `html: true` in it.

Reply:
Well in this case it is a attribute, so it MUST use escape anyways. It should be added back anyways.

What about this PR itself? Will it be merged?